user/homework.go

```diff
@@ -57,6 +59,36 @@ func (hw *Homework) Save() error {
 	return nil
 }
 
+// SaveAfterFetch filters out problems for tracks the user has yet to start
+// before saving them all to disk.
+func (hw *Homework) SaveAfterFetch(numArgs int) error {
+	hw.RejectUnstartedTracks(numArgs)
+	if len(hw.Items) == 0 {
+		fmt.Println(`
+You have yet to start a language track!
+View all available language tracks with "exercism tracks"
+Fetch exercises for your first track with "exercism fetch LANGUAGE"`)
+	}
```

Comment:
I didn't really think that this warranted an error, since when an error is returned from this that will bubble up to a `log.Fatal` call and exit in that fashion. I thought that might be a little jarring for the user, so I decided to go this route. However, I'm happy to change this to an error if you think that's more intention revealing!